src/content/docs/query-your-data/explore-query-data/use-charts/nrql-predictions.mdx

```diff
@@ -37,129 +46,428 @@ For further details on pricing, please contact our sales team or refer to your O
+      ```sql
+      FROM Transaction SELECT count(*) WHERE error IS TRUE TIMESERIES SINCE 1 day ago UNTIL 6 hours ago PREDICT USING 2 days
+      ```
 
-      Example query with the `USING` keyword that will use 2 days of historical data:
+      Example query with the `USING` keyword with the default query window that will use 49 hours of historical data (2 days from the `USING` and 1 hour from the default query window):
       ```sql
       FROM Transaction SELECT count(*) WHERE error IS TRUE TIMESERIES PREDICT USING 2 days
       ```
-      </Collapser>
+    </Collapser>
+
     </CollapserGroup>
 
-Example query with the `PREDICT` clause specifying custom hyperparameters, and adding the `BY` and `USING` keywords:
 
-```sql
-FROM Transaction SELECT count(*) WHERE error IS TRUE TIMESERIES PREDICT holtwinters(seasonality: AUTO, alpha: 0.2) BY 1 hour USING 2 hours
-```
+## Algorithm [#algorithm]
+Currently, <DNT>**NRQL predictions**</DNT> only supports the <DNT>**[Holt-Winters](https://en.wikipedia.org/wiki/Exponential_smoothing#Triple_exponential_smoothing_(Holt_Winters))**</DNT> algorithm, also known as exponential smoothing, for both seasonal and non-seasonal time series. This is a commonly used and standard algorithm for forecasting and time series predictions. For non-seasonal time series, only the level (average value) and trend (average increase or decrease) are modeled.
 
-This query sets the seasonality to automatic detection and emphasizes historical data by adjusting the level smoothing factor. It predicts one hour into the future and trains the model using the query window length along with the previous two hours of historical data. The chart displays the customized prediction as follows:
+<img
+     title="Non-seasonal predictions"
+     alt="Non-seasonal predictions"
+     src="/images/nrql-prediction-nonseasonal.webp"
+/>
+<figcaption>
+  Predictions generated using <DNT>**NRQL Predictions**</DNT> for a non-seasonal time series.
+</figcaption>
 
-   <img
-     title="Custom prediction"
-     alt="Custom prediction"
-     src="/images/nrql-prediction-custom.webp"
-   />
+For sesaonal time series, the seasonality is included in addition to the level and the trend. Seasonality represents the repeatable pattern that occurs on some periodic basis. We support automatic season detection of hourly, daily, and weekly seasonal lengths by default for the Holt-Winters model. In order to detect each seasonal length, you must ensure that you are providing enough data through a combination of the query window duration (specified using the `SINCE` and `UNTIL` clauses) and the `USING` clause. The minimum amount of data required is for each seasonal length is specified below. By default, when no `USING` clause is used, <DNT>**NRQL predictions**</DNT> will use <DNT>**3 times the query window**</DNT> to generate predictions.
```

Comment:
Typo in "sesaonal"; extra "is" in "The minimum amount of data required is for each..."